atc/db/build_event_source.go

```diff
@@ -121,12 +122,20 @@ func (source *buildEventSource) collectEvents(from uint) {
 			return
 		}
 
-		rows, err := psql.Select("event_id", "type", "version", "payload").
-			From(source.table).
-			Where(sq.Or{
+		eventsQuery := psql.Select("event_id", "type", "version", "payload").
+			From(source.table)
+
+		var query sq.SelectBuilder
+		if source.buildID > lock.MaxInt {
```

Comment:
As the other comment, `lock.MaxInt` is not needed, we can just use `math.MaxInt32` here.